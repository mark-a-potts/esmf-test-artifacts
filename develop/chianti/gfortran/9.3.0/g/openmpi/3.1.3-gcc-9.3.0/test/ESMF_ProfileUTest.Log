build time -- 2021-04-23 12:43:04.760193
20210423 124743.941 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124743.941 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124743.941 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124743.941 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124743.941 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124743.941 INFO             PET0 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124743.941 INFO             PET0 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124743.941 INFO             PET0 ESMF_COMM                   : openmpi
20210423 124743.941 INFO             PET0 ESMF_MOAB                   : enabled
20210423 124743.941 INFO             PET0 ESMF_LAPACK                 : enabled
20210423 124743.941 INFO             PET0 ESMF_NETCDF                 : enabled
20210423 124743.941 INFO             PET0 ESMF_PNETCDF                : disabled
20210423 124743.941 INFO             PET0 ESMF_PIO                    : enabled
20210423 124743.941 INFO             PET0 ESMF_YAMLCPP                : enabled
20210423 124743.941 INFO             PET0 ESMF Profiling Enabled
20210423 124743.941 INFO             PET0 ESMF Trace/Profile clock: REALTIME
20210423 124743.973 INFO             PET0  Beginning Test, file ESMF_ProfileUTest.F90, line 64
20210423 124743.973 INFO             PET0  NUMBER_OF_PROCESSORS           8
20210423 124743.973 INFO             PET0  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
20210423 124743.973 INFO             PET0  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
20210423 124744.145 INFO             PET0  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
20210423 124744.146 INFO             PET0 **************** Region Timings *******************
20210423 124744.146 INFO             PET0 Region                Count  Total (s)   Self (s)    Mean (s)    Min (s)     Max (s)
20210423 124744.146 INFO             PET0   [testcomp] Run 1    100    0.0001      0.0001      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET0     my_run            100    0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET0   reg1                1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET0   [testcomp] Init 1   1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET0   [testcomp] Final 1  1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET0  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
20210423 124744.146 INFO             PET0  Ending Test, file ESMF_ProfileUTest.F90, line 138
20210423 124744.146 INFO             PET0 Finalizing ESMF
20210423 124743.957 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.957 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124743.957 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124743.957 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124743.957 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124743.957 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.957 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124743.957 INFO             PET1 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124743.957 INFO             PET1 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124743.957 INFO             PET1 ESMF_COMM                   : openmpi
20210423 124743.957 INFO             PET1 ESMF_MOAB                   : enabled
20210423 124743.957 INFO             PET1 ESMF_LAPACK                 : enabled
20210423 124743.957 INFO             PET1 ESMF_NETCDF                 : enabled
20210423 124743.957 INFO             PET1 ESMF_PNETCDF                : disabled
20210423 124743.957 INFO             PET1 ESMF_PIO                    : enabled
20210423 124743.957 INFO             PET1 ESMF_YAMLCPP                : enabled
20210423 124743.957 INFO             PET1 ESMF Profiling Enabled
20210423 124743.957 INFO             PET1 ESMF Trace/Profile clock: REALTIME
20210423 124743.989 INFO             PET1  Beginning Test, file ESMF_ProfileUTest.F90, line 64
20210423 124743.989 INFO             PET1  NUMBER_OF_PROCESSORS           8
20210423 124743.989 INFO             PET1  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
20210423 124743.989 INFO             PET1  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
20210423 124744.145 INFO             PET1  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
20210423 124744.146 INFO             PET1 **************** Region Timings *******************
20210423 124744.146 INFO             PET1 Region                Count  Total (s)   Self (s)    Mean (s)    Min (s)     Max (s)
20210423 124744.146 INFO             PET1   [testcomp] Run 1    100    0.0001      0.0001      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET1     my_run            100    0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET1   reg1                1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET1   [testcomp] Init 1   1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET1   [testcomp] Final 1  1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET1  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
20210423 124744.146 INFO             PET1  Ending Test, file ESMF_ProfileUTest.F90, line 138
20210423 124744.146 INFO             PET1 Finalizing ESMF
20210423 124743.941 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124743.941 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124743.941 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124743.941 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124743.941 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124743.941 INFO             PET2 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124743.941 INFO             PET2 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124743.941 INFO             PET2 ESMF_COMM                   : openmpi
20210423 124743.941 INFO             PET2 ESMF_MOAB                   : enabled
20210423 124743.941 INFO             PET2 ESMF_LAPACK                 : enabled
20210423 124743.941 INFO             PET2 ESMF_NETCDF                 : enabled
20210423 124743.941 INFO             PET2 ESMF_PNETCDF                : disabled
20210423 124743.941 INFO             PET2 ESMF_PIO                    : enabled
20210423 124743.941 INFO             PET2 ESMF_YAMLCPP                : enabled
20210423 124743.941 INFO             PET2 ESMF Profiling Enabled
20210423 124743.941 INFO             PET2 ESMF Trace/Profile clock: REALTIME
20210423 124743.973 INFO             PET2  Beginning Test, file ESMF_ProfileUTest.F90, line 64
20210423 124743.973 INFO             PET2  NUMBER_OF_PROCESSORS           8
20210423 124743.973 INFO             PET2  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
20210423 124743.973 INFO             PET2  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
20210423 124744.145 INFO             PET2  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
20210423 124744.146 INFO             PET2 **************** Region Timings *******************
20210423 124744.146 INFO             PET2 Region                Count  Total (s)   Self (s)    Mean (s)    Min (s)     Max (s)
20210423 124744.146 INFO             PET2   [testcomp] Run 1    100    0.0001      0.0001      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET2     my_run            100    0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET2   reg1                1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET2   [testcomp] Init 1   1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET2   [testcomp] Final 1  1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET2  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
20210423 124744.146 INFO             PET2  Ending Test, file ESMF_ProfileUTest.F90, line 138
20210423 124744.146 INFO             PET2 Finalizing ESMF
20210423 124743.941 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124743.941 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124743.941 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124743.941 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124743.941 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124743.941 INFO             PET3 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124743.941 INFO             PET3 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124743.941 INFO             PET3 ESMF_COMM                   : openmpi
20210423 124743.941 INFO             PET3 ESMF_MOAB                   : enabled
20210423 124743.941 INFO             PET3 ESMF_LAPACK                 : enabled
20210423 124743.941 INFO             PET3 ESMF_NETCDF                 : enabled
20210423 124743.941 INFO             PET3 ESMF_PNETCDF                : disabled
20210423 124743.941 INFO             PET3 ESMF_PIO                    : enabled
20210423 124743.941 INFO             PET3 ESMF_YAMLCPP                : enabled
20210423 124743.941 INFO             PET3 ESMF Profiling Enabled
20210423 124743.941 INFO             PET3 ESMF Trace/Profile clock: REALTIME
20210423 124743.973 INFO             PET3  Beginning Test, file ESMF_ProfileUTest.F90, line 64
20210423 124743.973 INFO             PET3  NUMBER_OF_PROCESSORS           8
20210423 124743.973 INFO             PET3  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
20210423 124743.973 INFO             PET3  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
20210423 124744.145 INFO             PET3  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
20210423 124744.146 INFO             PET3 **************** Region Timings *******************
20210423 124744.146 INFO             PET3 Region                Count  Total (s)   Self (s)    Mean (s)    Min (s)     Max (s)
20210423 124744.146 INFO             PET3   [testcomp] Run 1    100    0.0001      0.0001      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET3     my_run            100    0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET3   [testcomp] Final 1  1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET3   reg1                1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET3   [testcomp] Init 1   1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET3  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
20210423 124744.146 INFO             PET3  Ending Test, file ESMF_ProfileUTest.F90, line 138
20210423 124744.146 INFO             PET3 Finalizing ESMF
20210423 124743.941 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET4 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124743.941 INFO             PET4 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124743.941 INFO             PET4 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124743.941 INFO             PET4 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124743.941 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET4 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124743.941 INFO             PET4 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124743.941 INFO             PET4 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124743.941 INFO             PET4 ESMF_COMM                   : openmpi
20210423 124743.941 INFO             PET4 ESMF_MOAB                   : enabled
20210423 124743.941 INFO             PET4 ESMF_LAPACK                 : enabled
20210423 124743.941 INFO             PET4 ESMF_NETCDF                 : enabled
20210423 124743.941 INFO             PET4 ESMF_PNETCDF                : disabled
20210423 124743.941 INFO             PET4 ESMF_PIO                    : enabled
20210423 124743.941 INFO             PET4 ESMF_YAMLCPP                : enabled
20210423 124743.941 INFO             PET4 ESMF Profiling Enabled
20210423 124743.941 INFO             PET4 ESMF Trace/Profile clock: REALTIME
20210423 124743.973 INFO             PET4  Beginning Test, file ESMF_ProfileUTest.F90, line 64
20210423 124743.973 INFO             PET4  NUMBER_OF_PROCESSORS           8
20210423 124743.973 INFO             PET4  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
20210423 124743.973 INFO             PET4  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
20210423 124744.145 INFO             PET4  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
20210423 124744.146 INFO             PET4 **************** Region Timings *******************
20210423 124744.146 INFO             PET4 Region                Count  Total (s)   Self (s)    Mean (s)    Min (s)     Max (s)
20210423 124744.146 INFO             PET4   [testcomp] Run 1    100    0.0001      0.0001      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET4     my_run            100    0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET4   reg1                1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET4   [testcomp] Init 1   1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET4   [testcomp] Final 1  1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET4  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
20210423 124744.146 INFO             PET4  Ending Test, file ESMF_ProfileUTest.F90, line 138
20210423 124744.146 INFO             PET4 Finalizing ESMF
20210423 124743.957 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.957 INFO             PET5 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124743.957 INFO             PET5 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124743.957 INFO             PET5 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124743.957 INFO             PET5 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124743.957 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.957 INFO             PET5 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124743.957 INFO             PET5 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124743.957 INFO             PET5 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124743.957 INFO             PET5 ESMF_COMM                   : openmpi
20210423 124743.957 INFO             PET5 ESMF_MOAB                   : enabled
20210423 124743.957 INFO             PET5 ESMF_LAPACK                 : enabled
20210423 124743.957 INFO             PET5 ESMF_NETCDF                 : enabled
20210423 124743.957 INFO             PET5 ESMF_PNETCDF                : disabled
20210423 124743.957 INFO             PET5 ESMF_PIO                    : enabled
20210423 124743.957 INFO             PET5 ESMF_YAMLCPP                : enabled
20210423 124743.957 INFO             PET5 ESMF Profiling Enabled
20210423 124743.957 INFO             PET5 ESMF Trace/Profile clock: REALTIME
20210423 124743.989 INFO             PET5  Beginning Test, file ESMF_ProfileUTest.F90, line 64
20210423 124743.989 INFO             PET5  NUMBER_OF_PROCESSORS           8
20210423 124743.989 INFO             PET5  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
20210423 124743.989 INFO             PET5  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
20210423 124744.145 INFO             PET5  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
20210423 124744.146 INFO             PET5 **************** Region Timings *******************
20210423 124744.146 INFO             PET5 Region                Count  Total (s)   Self (s)    Mean (s)    Min (s)     Max (s)
20210423 124744.146 INFO             PET5   [testcomp] Run 1    100    0.0001      0.0001      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET5     my_run            100    0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET5   [testcomp] Init 1   1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET5   reg1                1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET5   [testcomp] Final 1  1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET5  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
20210423 124744.146 INFO             PET5  Ending Test, file ESMF_ProfileUTest.F90, line 138
20210423 124744.146 INFO             PET5 Finalizing ESMF
20210423 124743.941 INFO             PET6 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET6 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124743.941 INFO             PET6 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124743.941 INFO             PET6 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124743.941 INFO             PET6 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124743.941 INFO             PET6 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET6 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124743.941 INFO             PET6 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124743.941 INFO             PET6 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124743.941 INFO             PET6 ESMF_COMM                   : openmpi
20210423 124743.941 INFO             PET6 ESMF_MOAB                   : enabled
20210423 124743.941 INFO             PET6 ESMF_LAPACK                 : enabled
20210423 124743.941 INFO             PET6 ESMF_NETCDF                 : enabled
20210423 124743.941 INFO             PET6 ESMF_PNETCDF                : disabled
20210423 124743.941 INFO             PET6 ESMF_PIO                    : enabled
20210423 124743.941 INFO             PET6 ESMF_YAMLCPP                : enabled
20210423 124743.941 INFO             PET6 ESMF Profiling Enabled
20210423 124743.941 INFO             PET6 ESMF Trace/Profile clock: REALTIME
20210423 124743.973 INFO             PET6  Beginning Test, file ESMF_ProfileUTest.F90, line 64
20210423 124743.973 INFO             PET6  NUMBER_OF_PROCESSORS           8
20210423 124743.973 INFO             PET6  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
20210423 124743.973 INFO             PET6  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
20210423 124744.146 INFO             PET6  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
20210423 124744.146 INFO             PET6 **************** Region Timings *******************
20210423 124744.146 INFO             PET6 Region                Count  Total (s)   Self (s)    Mean (s)    Min (s)     Max (s)
20210423 124744.146 INFO             PET6   [testcomp] Run 1    100    0.0001      0.0001      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET6     my_run            100    0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET6   [testcomp] Init 1   1      0.0001      0.0001      0.0001      0.0001      0.0001
20210423 124744.146 INFO             PET6   [testcomp] Final 1  1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET6   reg1                1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET6  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
20210423 124744.146 INFO             PET6  Ending Test, file ESMF_ProfileUTest.F90, line 138
20210423 124744.146 INFO             PET6 Finalizing ESMF
20210423 124743.941 INFO             PET7 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET7 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124743.941 INFO             PET7 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124743.941 INFO             PET7 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124743.941 INFO             PET7 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124743.941 INFO             PET7 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124743.941 INFO             PET7 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124743.941 INFO             PET7 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124743.941 INFO             PET7 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124743.941 INFO             PET7 ESMF_COMM                   : openmpi
20210423 124743.941 INFO             PET7 ESMF_MOAB                   : enabled
20210423 124743.941 INFO             PET7 ESMF_LAPACK                 : enabled
20210423 124743.941 INFO             PET7 ESMF_NETCDF                 : enabled
20210423 124743.941 INFO             PET7 ESMF_PNETCDF                : disabled
20210423 124743.941 INFO             PET7 ESMF_PIO                    : enabled
20210423 124743.941 INFO             PET7 ESMF_YAMLCPP                : enabled
20210423 124743.941 INFO             PET7 ESMF Profiling Enabled
20210423 124743.941 INFO             PET7 ESMF Trace/Profile clock: REALTIME
20210423 124743.973 INFO             PET7  Beginning Test, file ESMF_ProfileUTest.F90, line 64
20210423 124743.973 INFO             PET7  NUMBER_OF_PROCESSORS           8
20210423 124743.973 INFO             PET7  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
20210423 124743.973 INFO             PET7  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
20210423 124744.145 INFO             PET7  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
20210423 124744.146 INFO             PET7 **************** Region Timings *******************
20210423 124744.146 INFO             PET7 Region                Count  Total (s)   Self (s)    Mean (s)    Min (s)     Max (s)
20210423 124744.146 INFO             PET7   [testcomp] Run 1    100    0.0001      0.0001      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET7     my_run            100    0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET7   reg1                1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET7   [testcomp] Init 1   1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET7   [testcomp] Final 1  1      0.0000      0.0000      0.0000      0.0000      0.0000
20210423 124744.146 INFO             PET7  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
20210423 124744.146 INFO             PET7  Ending Test, file ESMF_ProfileUTest.F90, line 138
20210423 124744.146 INFO             PET7 Finalizing ESMF
