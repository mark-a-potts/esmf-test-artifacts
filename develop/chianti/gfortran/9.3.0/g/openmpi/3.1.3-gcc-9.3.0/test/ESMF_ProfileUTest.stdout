build time -- 2021-04-23 12:43:04.760193
  Beginning Test, file ESMF_ProfileUTest.F90, line 64
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_ProfileUTest.F90, line 64
  Beginning Test, file ESMF_ProfileUTest.F90, line 64
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_ProfileUTest.F90, line 64
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_ProfileUTest.F90, line 64
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_ProfileUTest.F90, line 64
  NUMBER_OF_PROCESSORS           8
  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
  NUMBER_OF_PROCESSORS           8
  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
  Beginning Test, file ESMF_ProfileUTest.F90, line 64
  Beginning Test, file ESMF_ProfileUTest.F90, line 64
  NUMBER_OF_PROCESSORS           8
  NUMBER_OF_PROCESSORS           8
  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
  PASS  Test trace user region enter, ESMF_ProfileUTest.F90, line 80
  PASS  Test trace user region exit, ESMF_ProfileUTest.F90, line 88
 Inside Init
 Inside Init
 Inside Init
 Inside Init
 Inside Init
 Inside Init
 Inside Init
 Inside Init
 Inside Finalize
 Inside Finalize
 Inside Finalize
 Inside Finalize
 Inside Finalize
 Inside Finalize
 Inside Finalize
 Inside Finalize
  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
  PASS  Test trace phase enter/exit, ESMF_ProfileUTest.F90, line 116
  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
  Ending Test, file ESMF_ProfileUTest.F90, line 138
  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
  Ending Test, file ESMF_ProfileUTest.F90, line 138
  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
  Ending Test, file ESMF_ProfileUTest.F90, line 138
  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
  Ending Test, file ESMF_ProfileUTest.F90, line 138
  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
  Ending Test, file ESMF_ProfileUTest.F90, line 138
  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
  Ending Test, file ESMF_ProfileUTest.F90, line 138
  Ending Test, file ESMF_ProfileUTest.F90, line 138
  PASS  Close trace, ESMF_ProfileUTest.F90, line 127
  Ending Test, file ESMF_ProfileUTest.F90, line 138
  PET           7  Test Elapsed Time    172.98000000000002       msec.
  PET           6  Test Elapsed Time    173.95700000000002       msec.
  PET           5  Test Elapsed Time    94.763000000000005       msec.
  PET           3  Test Elapsed Time    142.58799999999999       msec.
  PET           4  Test Elapsed Time    174.55300000000000       msec.
  PET           2  Test Elapsed Time    174.54999999999998       msec.
  PET           1  Test Elapsed Time    98.364999999999981       msec.
  PET           0  Test Elapsed Time    174.66499999999999       msec.
